src/dmd/frontend.d

```diff
@@ -101,9 +101,15 @@ Returns: full path to the found `dmd.conf`, `null` otherwise.
 string findDMDConfig(const(char)[] dmdFilePath)
 {
     import dmd.dinifile : findConfFile;
-    import std.string : fromStringz, toStringz;
 
-    return findConfFile(dmdFilePath, "dmd.conf").idup;
+    version (Posix)
+        enum configFile = "dmd.conf";
+    else version (Windows)
+        enum configFile = "sc.ini";
+    else
+        static assert(false, "Not implemented for this platform");
```

Comment:
Id' go with `sc.ini` on `Windows` and default to `dmd.conf` instead, no `static assert` needed.

Reply:
Fine.

Reply:
Done.